### PR TITLE
Bug 2132733: Display raw-capacity for non-standard deployments

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@SanjalKatiyar Please review

File: packages/odf/modals/add-capacity/add-capacity-modal.tsx [hunk 1]
```diff
@@ -224,7 +224,9 @@ export const AddCapacityModal: React.FC<AddCapacityModalProps> = ({
 
   const deviceSets: DeviceSet[] = ocsConfig?.spec?.storageDeviceSets || [];
   const osdSizeWithUnit = getRequestedPVCSize(deviceSets?.[0]?.dataPVCTemplate);
-  const osdSizeWithoutUnit: number = OSD_CAPACITY_SIZES[osdSizeWithUnit];
+  // ODF support Gi and Ti for any custome size
+  const [osdSize, unit] = osdSizeWithUnit.split(/(\d+)/).filter(Boolean);
```
Comment (bipuladh) [inline packages/odf/modals/add-capacity/add-capacity-modal.tsx:228]: nice!

**Review by bipuladh — APPROVED** [review]

File: packages/odf/modals/add-capacity/add-capacity-modal.tsx (hunk 1 above)
Comment (TimothyAsirJeyasing) [reply packages/odf/modals/add-capacity/add-capacity-modal.tsx:228]: Thank you

**Comment by SanjalKatiyar** [comment]
PR is merged, but we still need acks on the BZ and get target release set to 4.13.0
BZ is still in POST state...

**Pull request merged**